### PR TITLE
Small CSS fixes

### PR DESCRIPTION
**Pull request by fritzSF**

### CODE REVIEW

File: app/styles/_helpers.scss [hunk 1]
```diff
@@ -266,9 +266,56 @@ time {
...
-}
\ No newline at end of file
+}
+
+
+// Common margin overrides
+// --------------------------------------------------
+
+// top
+.sd-margin-t--1 {
```
Comment (tomaskikutis) [inline app/styles/_helpers.scss]: What kind of design are you doing so you need such rules? :)

Comment (fritzSF) [reply app/styles/_helpers.scss]: Overrides for certain elements, to avoid creating to much modifiers for each element.

Comment (tomaskikutis) [reply app/styles/_helpers.scss]: Can you show an example of a design where you would put the override otherwise?

Comment (fritzSF) [reply app/styles/_helpers.scss]: Usage should be restricted, I prefer modifiers, but for certain elements we end up needing too much options. This is for such cases, I would avoid using it to much

**Pull request merged**